### ISSUE TITLE
Traffic Sign Recognition

### ISSUE THREAD

**Issue by Sindhu-2004** [abhisheks008/DL-Simplified#597]
Implementation of  Traffic Sign recognition using a Convolutional Neural Network to address the increased risk of road accidents caused by poor visibility of traffic signs especially in winter, which is crucial for the safety of Automatic Driving Vehicles.

**Comment by abhisheks008** [comment]
Can you share the dataset and approach for solving this issue?

**Comment by Sindhu-2004** [comment]
Title : Traffic Sign Recognition
Aim : Implementation of Traffic Sign recognition using a Convolutional Neural Network to address the increased risk of road accidents caused by poor visibility of traffic signs especially in winter, which is crucial for the safety of Automatic Driving Vehicles.
Dataset : 
[Traffic Sign ] https://www.kaggle.com/datasets/ahemateja19bec1025/traffic-sign-dataset-classification
Approach : I will be implementing Convolutional Neural Network, Keras

**Comment by abhisheks008** [comment]
One issue at a time.

**Comment by kyra-09** [comment]
Hey @abhisheks008  Can you assign this issue to me?

**Comment by abhisheks008** [comment]
Hi @Sindhu-2004 as you have opened various new issues for this repo. If I assign someone in this issue, will that be okay with you?

**Comment by Sindhu-2004** [comment]
I am ok if you assign this traffic sign recognition issue with others, but I wanted to work remaining other issues which I have opened

**Comment by kyra-09** [comment]
Then if it is okay then @abhisheks008 please assign me this issue.

**Comment by abhisheks008** [comment]
Hi @kyra-09 can you please share the detailed approach for solving this issue?

**Comment by kyra-09** [comment]
Full Name :- Aaradhya Singh
Github Id :- https://github.com/kyra-09
Email Id :- aaradhyasinghgaur@gmail.com
Aim :- Traffic Sign recognition using DL models
Approach :- I will use Transfer learning with already existing network architecture on a specified dataset including data analysis and comaprison between the performance and accuracy of  models used for the purpose .....I'll also optimize the models according to dimensions and annotations given for the dataset .
Tools I might use (depending upon the complexity of dataset) :- Keras , PyTorch , OpenCV
Program :- GSSOC-2024 Contributor

**Comment by abhisheks008** [comment]
Assigned @kyra-09
